ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -20,8 +20,11 @@
 tier2 = pytest.mark.tier2(value=2)
 tier3 = pytest.mark.tier3(value=3)
 tier4 = pytest.mark.tier4(value=4)
+tier4a = compose(tier4, pytest.mark.tier4a)
+tier4b = compose(tier4, pytest.mark.tier4b)
+tier4c = compose(tier4, pytest.mark.tier4c)
 
-tier_marks = [tier1, tier2, tier3, tier4]
+tier_marks = [tier1, tier2, tier3, tier4, tier4a, tier4b, tier4c]
```

Comment:
@petr-balogh should we remove 'tier4' from here so it won't be picked automatically when running all tiers?

Reply:
No shouldn't be removed, this list just represent all tier markers.